packages/web-app-admin-settings/src/components/Groups/SideBar/MembersPanel.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="oc-ml-s">
+    <oc-text-input
+      v-model="filterTerm"
+      class="oc-text-truncate oc-mr-s oc-mt-m"
+      :label="$gettext('Filter members')"
+    />
+    <div ref="membersListRef" data-testid="space-members">
+      <div v-if="!filteredGroupMembers.length">
+        <h3 class="oc-text-bold oc-text-medium" v-text="$gettext('No members found')" />
+      </div>
+      <div v-if="filteredGroupMembers.length" class="oc-mb-m" data-testid="group-members">
+        <h3 class="oc-text-bold oc-text-medium" v-text="$gettext('Members')" />
+        <members-role-section :group-members="filteredGroupMembers" />
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { computed, defineComponent, inject, ref, watch, unref } from 'vue'
+import MembersRoleSection from '../../Groups/SideBar/MembersRoleSection.vue'
+import Fuse from 'fuse.js'
+import Mark from 'mark.js'
+import { Group } from 'web-client/src/generated'
+
+export default defineComponent({
+  name: 'GroupsMembersPanel',
+  components: { MembersRoleSection },
+  setup() {
+    const group = inject<Group>('group')
```

Comment:
I noticed recently that most of our `inject` types are wrong. When we pass them as `computed` or as `ref`, they are a `Ref`. 

```suggestion
    const group = inject<Ref<Group>>('group')
```